docs/interactions/Message_Components.md

```diff
@@ -15,20 +15,19 @@ The top-level `components` field is an array of [Action Row](#DOCS_INTERACTIONS_
-| ------------ | ----------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------ |
+|--------------|-------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------|
 | type         | integer                                                                                                     | [component type](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/component-object-component-types)  | all types                                                                                                                            |
 | custom_id?   | string                                                                                                      | a developer-defined identifier for the component, max 100 characters                      | [Buttons](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/buttons), [Select Menus](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menus)         |
 | disabled?    | boolean                                                                                                     | whether the component is disabled, default `false`                                        | [Buttons](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/buttons), [Select Menus](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menus)         |
 | style?       | integer                                                                                                     | one of [button styles](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/button-object-button-styles) | [Buttons](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/buttons)                                                                             |
-| label?       | string                                                                                                      | text that appears on the button, max 80 characters                                        | [Buttons](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/buttons)                                                                             |
+| label?       | string                                                                                                      | text that appears on the button, max 80 characters; the label for the text input          | [Buttons](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/buttons), [Text Inputs](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/text-inputs)           |
```

Comment:
Should just make label non optional and add footnote that it is not required in buttons.